SPECS/tpm2-tss/tpm2-tss.spec

```diff
@@ -1,94 +1,74 @@
 Summary:        OSS implementation of the TCG TPM2 Software Stack (TSS2)
 Name:           tpm2-tss
-Version:        2.4.6
-Release:        3%{?dist}
+Version:        4.0.1
+Release:        1%{?dist}
 License:        BSD
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System Environment/Security
 URL:            https://github.com/tpm2-software/tpm2-tss
-Source0:        https://github.com/tpm2-software/tpm2-tss/releases/download/%{version}/%{name}-%{version}.tar.gz
-Patch0:         CVE-2023-22745.patch
+
+Source0: https://github.com/tpm2-software/tpm2-tss/releases/download/%{version}/%{name}-%{version}.tar.gz
+Source1: %{name}.sysusers
 BuildRequires:  json-c-devel
```

Comment:
it looks like json-c-devel isn't in photon spec, is this removable or do we need it?

Reply:
it came from the fedora 4.0.1 spec .. i think it must've been required.   hard to imagine that i would import that alone from fedora.

Reply:
without this Requires:

```
time="2024-01-30T23:09:27Z" level=debug msg="checking for JSONC... no"
time="2024-01-30T23:09:27Z" level=debug msg="configure: error: Package requirements (json-c) were not met:"
time="2024-01-30T23:09:27Z" level=debug
time="2024-01-30T23:09:27Z" level=debug msg="Package 'json-c', required by 'virtual:world', not found"
```

will add comment